packages/core/src/resources/MergeRequestApprovals.ts

```diff
@@ -57,25 +95,27 @@ export class MergeRequestApprovals<C extends boolean = false> extends BaseResour
     } else {
       url = `projects/${pId}/approval_rules`;
     }
-    return RequestHelper.get()(this, url, options);
+    return RequestHelper.get<MergeRequestApprovalRuleSchema[]>()(this, url, options);
   }
 
-  approvals(
-    projectId: string | number,
-    { mergerequestIid, ...options }: { mergerequestIid?: number } & BaseRequestOptions = {},
-  ) {
+  approvalConfig(projectId: string | number, options?: BaseRequestOptions) {
```

Comment:
This is a good change, def improved readability. The only caveat is that it may be misleading for those looking for the approvalConfig for merge requests. If you looked at the docs you'd assume a similar API, but its a different function entirely. I think we could still do something similar to what was done before, but with a inferred response based on the input argument.  I know i do it somewhere else in the codebase, but id have to double check and follow up!

Reply:
Ok interesting, if you point me to that i'll take a look